src/controllers/cart.controller.ts

```diff
@@ -0,0 +1,163 @@
+import { Request, Response } from "express";
+import { CreateCartAttributes, CreateCartItemAttributes } from "../types/cart.types";
+/* import Cart from "../database/models/cart.model";  */
+import { CartService } from "../services/cart.services";
+import { ProductService } from "../services/Product.services";
+
+
+export const createCart = async (req: Request, res: Response) => {
+    try {
+      const { name, description, userId } = req.body; 
```

Comment:
userId must come from req.user instead req.body

Reply:
I have resolved it you can check